_posts/2013-08-28-odataquerybuilder-a-cross-browser-javascript-library-for-building-odata-queries-3.html

```diff
@@ -32,7 +32,7 @@ <h2>The problem: generating syntactically correct OData queries and interpreting
 <p><a href="https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate%20le%20datetime'2013-08-07'%20and%20substringof(ShipAddress,%20'12')%20eq%20true">https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate le datetime'2013-08-07' and substringof(ShipAddress, '12') eq true</a></p>
 <p>The service returns an empty list:</p>
 <p>[gist id="261b948a9d18562d600e" file="EmptyList.xml"]</p>
-<p>The list is empty because the order of the parameters for substringof was incorrect (see <a href="https://www.odata.org/documentation/odata-v2-documentation/uri-conventions/">https://www.odata.org/documentation/odata-v2-documentation/uri-conventions/</a>).  The correct URL should in fact be <a href="https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate%20le%20datetime'2013-08-07'%20and%20substringof('12',%20ShipAddress)%20eq%20true">https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate le datetime'2013-08-07' and substringof('12', ShipAddress) eq true</a> which yields all the orders (click the URL to see them all).</p>
+<p>The list is empty because the order of the parameters for substringof was incorrect (see <a href="/documentation/odata-version-2-0/uri-conventions/">/documentation/odata-version-2-0/uri-conventions/</a>).  The correct URL should in fact be <a href="https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate%20le%20datetime'2013-08-07'%20and%20substringof('12',%20ShipAddress)%20eq%20true">https://services.odata.org/Northwind/Northwind.svc/Orders?$filter=OrderDate le datetime'2013-08-07' and substringof('12', ShipAddress) eq true</a> which yields all the orders (click the URL to see them all).</p>
```

Comment:
Since the link text here contains the actual URL, I wonder whether it would make sense to display an absolute URL here (just for the text content, not the `href` attribute) and use the `{{ | preprend site.baseurl }}` trick to display the correct base URL. Just in case the user selects the text to copy-pase it, it would be great if they paste something that can be navigated to directly when outside this page.